docs/accordion.component.md

```diff
@@ -29,15 +29,6 @@ export class MyComponent implements OnInit {
 }
 ```
```

Comment:
Why this table has been removed?

Reply:
The properties are actually declared in Accordion Group, which has its own separate page and the table is OK there. Having the pages separate like this is useful for auto-generating the props tables (because the .ts file corresponds directly to the .md file for the same class). However, if you think this is unhelpful for the reader then I'll try to find a way around it.